### ISSUE TITLE
Running pip install -U rtichoke does not install anything.

### ISSUE THREAD

**Issue by eddieimada** [randy3k/rtichoke#56]
I tried installing rtichoke via pip, but even though the installation succeeds it is not found when I call rtichoke. 

> ➜  ~ pip install -U rtichoke     
Collecting rtichoke
Collecting rapi>=0.0.16 (from rtichoke)
Collecting lineedit>=0.0.5 (from rtichoke)
Collecting multipledispatch (from rapi>=0.0.16->rtichoke)
  Using cached https://files.pythonhosted.org/packages/d1/0d/0d51971308af74de1a3a66e52cb518cb14c3fa9b0c6736105eacf77d2d09/multipledispatch-0.5.0-py2.py3-none-any.whl
Collecting pygments (from lineedit>=0.0.5->rtichoke)
  Using cached https://files.pythonhosted.org/packages/02/ee/b6e02dc6529e82b75bb06823ff7d005b141037cb1416b10c6f00fc419dca/Pygments-2.2.0-py2.py3-none-any.whl
Collecting six (from lineedit>=0.0.5->rtichoke)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting wcwidth (from lineedit>=0.0.5->rtichoke)
  Using cached https://files.pythonhosted.org/packages/7e/9f/526a6947247599b084ee5232e4f9190a38f398d7300d866af3ab571a5bfe/wcwidth-0.1.7-py2.py3-none-any.whl
Installing collected packages: multipledispatch, rapi, pygments, six, wcwidth, lineedit, rtichoke
Successfully installed lineedit-0.0.5 multipledispatch-0.5.0 pygments-2.2.0 rapi-0.0.16 rtichoke-0.2.1 six-1.11.0 wcwidth-0.1.7
➜  ~ rtichoke                    
zsh: command not found: rtichoke

**Comment by randy3k** [comment]
It looks like your pip is linking to a different python binary as your global python.

Try `python -m pip install rtichoke -U`?

**Comment by eddieimada** [comment]
Just tried, it still the same. It does not find rtichoke. Any suggestion?

**Comment by randy3k** [comment]
would you check where `python` and `pip` are located? Their locations may give us some hints.

**Comment by emilBeBri** [comment]
It's the same for me, I'm on a Lubuntu 18.04 machine. I don't really know python so just installing as per instructions with:

`pip install -U rtichoke`

The output looks like this:
```
Collecting rtichoke
Collecting lineedit>=0.1.1 (from rtichoke)
Collecting rapi>=0.1.2 (from rtichoke)
Collecting pygments (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/02/ee/b6e02dc6529e82b75bb06823ff7d005b141037cb1416b10c6f00fc419dca/Pygments-2.2.0-py2.py3-none-any.whl
Collecting wcwidth (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/7e/9f/526a6947247599b084ee5232e4f9190a38f398d7300d866af3ab571a5bfe/wcwidth-0.1.7-py2.py3-none-any.whl
Collecting six>=1.9.0 (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting multipledispatch (from rapi>=0.1.2->rtichoke)
  Using cached https://files.pythonhosted.org/packages/3d/a3/3638c2232eb513a9f876bb96e2e400f18d2f5bdc2e7abe84194c8bd38c2d/multipledispatch-0.6.0-py2-none-any.whl
Installing collected packages: pygments, wcwidth, six, lineedit, multipledispatch, rapi, rtichoke
Successfully installed lineedit-0.1.1 multipledispatch-0.6.0 pygments-2.2.0 rapi-0.1.2 rtichoke-0.2.12 six-1.11.0 wcwidth-0.1.7

```
also tried the suggestion:

`python -m pip install rtichoke -U`

which gives this output:
```
Collecting rtichoke
Collecting lineedit>=0.1.1 (from rtichoke)
Collecting rapi>=0.1.2 (from rtichoke)
Collecting pygments (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/02/ee/b6e02dc6529e82b75bb06823ff7d005b141037cb1416b10c6f00fc419dca/Pygments-2.2.0-py2.py3-none-any.whl
Collecting wcwidth (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/7e/9f/526a6947247599b084ee5232e4f9190a38f398d7300d866af3ab571a5bfe/wcwidth-0.1.7-py2.py3-none-any.whl
Collecting six>=1.9.0 (from lineedit>=0.1.1->rtichoke)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting multipledispatch (from rapi>=0.1.2->rtichoke)
  Using cached https://files.pythonhosted.org/packages/3d/a3/3638c2232eb513a9f876bb96e2e400f18d2f5bdc2e7abe84194c8bd38c2d/multipledispatch-0.6.0-py2-none-any.whl
Installing collected packages: pygments, wcwidth, six, lineedit, multipledispatch, rapi, rtichoke
Successfully installed lineedit-0.1.1 multipledispatch-0.6.0 pygments-2.2.0 rapi-0.1.2 rtichoke-0.2.12 six-1.11.0 wcwidth-0.1.7
```
I have both python 2.7.14 and python 3.6 installed, seems to happen easily in Ubuntu (if I try to go for a complete removal of any of them, in synaptic package manager, it warns me that it will also remove stuff like firefox, so...) 

how to proceed from here? Thank you!

**Comment by randy3k** [comment]
Maybe this post will help

https://stackoverflow.com/questions/35898734/pip-installs-packages-successfully-but-executables-not-found-from-command-line

**Comment by emilBeBri** [comment]
Yes that helped me in the right direction. It's because the python modules (I think they're called?) aren't in the $PATH. You can add the directory like this:

export PATH=$PATH:/home/emil/.local/bin/

where "emil" is my home folder.

This won't last after a restart. add it permanently by editing the .profile-file:

nano /home/emil/.profile

and add the abovementioned command. 

Thank you! looking forward to trying out rtichoke. Been a happy user of R-Box for a long time, time to upgrade!

**Comment by scm-unimelb** [comment]
is this issue I just closed any help: https://github.com/randy3k/rtichoke/issues/62
